src/components/Header.tsx

```diff
@@ -7,6 +7,8 @@ export interface HeaderProps {
   alt?: string;
   logOut?: string;
   loggedIn?: boolean;
+  /** This is expected to be a ul element or a div with an ul element. */
+  nav?: JSX.Element;
 }
```

Comment:
Why not name it something more generic, like `content`, and maybe let it accept an array of elements, so that you can use it for other things too?  (Like if you wanted an separate piece of text for something, or a button other than the log out one.)

Reply:
I think that would complicate this basic header. This is also specifically for the navigation and you wouldn't have extra text or a button in that element.